### ISSUE TITLE
Cant to add data in the google sheet

### ISSUE THREAD

**Issue by maxmenGMK** [sammachin/node-red-contrib-google-sheets#12]
It is more "example or manual absent" than real issue. I can to read data with this node from gsheet perfectly but i cant to find the way to write there. I was try to use "append row" method and to put nessesary data to topic [["A4","B4","C4"]] and tried another syntax but there still no new rows with data in the sheet. Where i can get the example how i should to write data in cells?

**Comment by Jacksonbm1** [comment]
I agree, there was almost no way I would have figured this out without you saying this, but also, it should be in JSON (at least based on my experiments). Like whatever you submit should be JSON
